src/components/autofillFields/AutofillFields.tsx

```diff
@@ -1,20 +1,163 @@
+import { useAppState } from '@/hooks/useAppState'
+import { IClient } from '@/types/interfaces'
+import { staticAutofillValues } from '@/utils/constants'
+import { TiptapEditorUtils } from '@/utils/tiptapEditorUtils'
+import { Editor } from '@tiptap/react'
+import { When } from '../hoc/When'
+import { useEffect, useState } from 'react'
+
 const AutofillFields = () => {
+  const appState = useAppState()
+
+  const tiptapEditorUtils = new TiptapEditorUtils(
+    appState?.appState.editor as Editor,
+  )
+
+  useEffect(() => {
+    if (!appState?.appState.selectedClient && !appState?.appState.readOnly)
+      return
+    appState?.setClientCompanyName('')
+    ;(async () => {
```

Comment:
`;`

Reply:
prettier automatically puts this here @saugatmaharjan